src/My.Extensions.Localization.Json/JsonStringLocalizer.cs

```diff
@@ -171,7 +171,7 @@ private void BuildResourcesCache(string culture)
                 if (File.Exists(_searchedLocation))
                 {
                     var builder = new ConfigurationBuilder()
-                    .SetBasePath(_resourcesPath)
+                    .SetBasePath(Path.Combine(Directory.GetCurrentDirectory(),_resourcesPath))
```

Comment:
We need to change this line instead

```csharp
_searchedLocation = Path.Combine(Directory.GetCurrentDirectory(), _resourcesPath, resourceFile);
```

Reply:
the above-mentioned line has no effect on the issue, I changed it in the constructor